nmsg_msgmod.pyx

```diff
@@ -14,6 +14,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+def msgmod_oname_to_oid(str oname):
+    cdef unsigned oid
+    cdef char *oname2
+    t = oname.encode('ascii')
+    oname2 = t
+    oid = nmsg_alias_by_value(nmsg_alias_operator, oname2)
+    if oid == 0:
+        raise Exception, 'unknown operator name: %s' % oname
+    return oid
+
+def msgmod_grname_to_grid(str grname):
+    cdef unsigned oid
+    cdef char *grname2
+    t = grname.encode('ascii')
+    grname2 = t
```

Comment:
Is the "t" variable necessary?  would oname2 = oname.encode('ascii') 
etc. work?

Reply:
I thought the same thing when I first saw that, but if I try to skip using the "t" Python variable I get this error:

```
Error compiling Cython file:
------------------------------------------------------------
...

def msgmod_oname_to_oid(str oname):
    cdef unsigned oid
    cdef char *oname2
    # t = oname.encode('ascii')
    oname2 = oname.encode('ascii')
                        ^
------------------------------------------------------------

nmsg_msgmod.pyx:21:25: Storing unsafe C derivative of temporary Python reference
```

Reply:
I figured that might be the case.  Maybe add a comment next time you are editing that file?